mist/worker/src/main/scala/io/hydrosphere/mist/worker/WorkerActor.scala

```diff
@@ -65,14 +82,12 @@ class SharedWorkerActor(
   }
 
   override def receive: Receive = {
-    case req @ RunJobRequest(id, params) =>
+    case req@RunJobRequest(id, params) =>
```

Comment:
Don't you like spaces?)

Reply:
I think it automatically indents it like this :)